Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.Shared/Controls/BocAutoCompleteReferenceValueWebService.asmx.cs

```diff
@@ -77,6 +78,17 @@ public IconProxy GetIcon (string businessObjectClass, string businessObject, str
       if (searchString == "throw")
         throw new InvalidOperationException("I'm always going to throw an exception if you search for 'throw'!");
 
+      if (searchString == "testlist")
+      {
+        var count = completionSetCount ?? 10;
+        var resultItems = Enumerable.Range(0, 20)
+            .Skip(completionSetOffset)
+            .Take(count)
+            .Select(e => new BusinessObjectWithIdentityProxy { DisplayName = $"Person {e}", UniqueIdentifier = e.ToString() })
```

Comment:
Recommendation: BocAutoComplete works best when the items match the search string. So, prefix with "testlist "?